warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 21534)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 22591)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 4014)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 4023)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 4030)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 4046)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 4080)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 4095)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 4106)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 4129)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 4144)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 6177)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 6188)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 6195)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 6211)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 6229)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 6246)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 6264)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 6277)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 6290)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 6555)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 6566)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 6583)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 6592)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 6615)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 6627)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 6642)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 6659)
warning: Debugger speedups using cython not found. Run '"/usr/bin/python3" "/home/user/.p2/pool/plugins/org.python.pydev.core_7.5.0.202001101138/pysrc/setup_cython.py" build_ext --inplace' to build.
pydev debugger: starting (pid: 6680)
